src/service/resources-impl.js

```diff
@@ -207,14 +214,8 @@ export class ResourcesImpl {
         : null);
       try {
         this.intersectionObserver_ = new IntersectionObserver(
-          this.intersects_.bind(this),
-          {
-            root,
-            // TODO(willchou): Is 3x viewport loading rectangle too large given that
```

Comment:
why remove this TODO? Do we no longer want to separately prioritize 1vp and 3vp?